testsuite/MDAnalysisTests/analysis/test_wbridge.py

```diff
@@ -17,6 +15,29 @@ def test_import_from_hbonds():
                              "importing WaterBridgeAnalysis from "
                              "MDAnalysis.analysis.hbonds failed.'")
 
+
+def test_import_warning():
+    with pytest.warns(DeprecationWarning) as record:
+        from MDAnalysis.analysis.hbonds.wbridge_analysis import (
+            WaterBridgeAnalysis, )
+    assert len(record) == 2
+    assert record[0].message.args[0] == 'This module is deprecated as of '\
+                                        'MDAnalysis version 1.0.It will be '\
+                                        'removed in MDAnalysis version 2.0'\
+                                        'Please use '\
+                                        'MDAnalysis.analysis.hydrogenbonds.'\
+                                        'hbond_analysis instead.'
+    assert record[1].message.args[0] == 'This module has been moved to '\
+                                        'MDAnalysis.analysis.hydrogenbondsIt '\
+                                        'will be removed in MDAnalysis '\
+                                        'version 2.0. Please use MDAnalysis.'\
+                                        'analysis.hydrogenbonds.'\
+                                        'wbridge_analysis instead.'
+
+
+from MDAnalysis.analysis.hydrogenbonds.wbridge_analysis import (
```

Comment:
Strange, does it not work if this import is at the top of the file?

Reply:
@IAlibay  The warning is only issued once when it is first imported. Importing it again won't give the same warning.

Reply:
I see, well as long as it works, it's a lot better than whatever I had to do for some of the other tests with import issues 😅

Reply:
@IAlibay but it creates PEP8 problem as the module-level import is not at the top of the file.

Reply:
Unfortunately I can't pull your PR (since it's on your develop branch), so I can't tell if there is some weird way to go around this, but if this is the only way I think we can live with breaking PEP8 a little bit (it's more guidelines rather than strict rules to live by).